e2e/groups/group-edit-requires-approval.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { test, expect } from '@playwright/test';
+import { initAsUsualUser } from '../helpers/e2e_auth';
+import { convertDateToString } from 'src/app/utils/input-date';
+import { DAYS } from 'src/app/utils/duration';
+
+test.describe.configure({ mode: 'serial' });
+
+test.use({
+  timezoneId: 'Africa/Accra',
+});
```

Comment:
I don't understand why. What was failing? If the website only works on UTC, that's a problem :-) so I would have had the tendency to avoid UTC for the tests

Reply:
Fixed